db/migrate/20180626125654_make_maintenance_zone_record.rb

```diff
@@ -0,0 +1,32 @@
+class MakeMaintenanceZoneRecord < ActiveRecord::Migration[5.0]
+  class Zone < ActiveRecord::Base
+    validates_presence_of   :name
+    validates_uniqueness_of :name
```

Comment:
These must be removed.  In a global region, since zones are transferred from other regions, there are multiple "default" zones, for example.  As such, this migration will fail to migrate a global database.